### PR TITLE
HTBHF-632 Added call to downstream dwp api.

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: api/src/main/resources/application.yml [hunk 1]
```diff
@@ -32,4 +32,9 @@ logging:
     console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n"
   level:
     ROOT: ${vcap.services.variable-service.credentials.dwp-api-app-root-loglevel:info}
-    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
\ No newline at end of file
+    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
+
+dwp:
+  eligibility-check-in-weeks: 4
+  monthly-income-threshold: 410.00
```
Comment (YetAnotherMatt) [inline api/src/main/resources/application.yml]: monthly-income-threshold is 408

File: api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityStatus.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+@JsonFormat(shape = JsonFormat.Shape.STRING)
+public enum EligibilityStatus {
```
Comment (YetAnotherMatt) [inline api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityStatus.java:11]: Can we have some javadoc explaining this enum?
e.g. "The possible states that a claim can be in according to the DWP", plus in-line comments about each one

Comment (dwybourn) [reply api/src/main/java/uk/gov/dhsc/htbhf/dwp/model/EligibilityStatus.java:11]: Updated class with javadoc.

File: api/src/main/resources/application.yml [hunk 3]
```diff
@@ -32,4 +32,9 @@ logging:
     console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n"
   level:
     ROOT: ${vcap.services.variable-service.credentials.dwp-api-app-root-loglevel:info}
-    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
\ No newline at end of file
+    uk.gov.dhsc: ${vcap.services.variable-service.credentials.dwp-api-app-loglevel:debug}
+
+dwp:
+  eligibility-check-in-weeks: 4
```
Comment (YetAnotherMatt) [inline api/src/main/resources/application.yml:38]: As discussed, we shouldn't be defining this in this repository - one for a future pr

**Review by YetAnotherMatt — APPROVED** [review]

**Pull request merged**